tests/integration/llm/sagemaker-endpoint-tests.py

```diff
@@ -26,13 +29,19 @@
         "payload": {
             "prompt": "A picture of a cowboy in space"
         },
-        "deserializer": sagemaker.deserializers.BytesDeserializer()
+        "deserializer": sagemaker.deserializers.BytesDeserializer(),
+        "metrics_config": {
```

Comment:
can we determine this from the class? after we create a model we can get the engine name like

`model.engine[0]`

then we can use a map between the engine name and the desired metric config engine (seems like we only really need  a mapping from python to accelerate)